src/spyglass/common/common_interval.py

```diff
@@ -51,11 +51,11 @@ def insert_from_nwbfile(cls, nwbf, *, nwb_file_name):
         for _, epoch_data in epochs.iterrows():
             epoch_dict = {
                 "nwb_file_name": nwb_file_name,
-                "interval_list_name": epoch_data[1].tags[0]
-                if epoch_data[1].tags
+                "interval_list_name": epoch_data.tags[0]
+                if epoch_data.tags
                 else f"interval_{epoch_data[0]}",
                 "valid_times": np.asarray(
-                    [[epoch_data[1].start_time, epoch_data[1].stop_time]]
+                    [[epoch_data.start_time, epoch_data.stop_time]]
```

Comment:
@samuelbray32 - You mentioned that this was related to #654. I don't think there are any changes there that impact how the epoch data is coded in the file. I'm thinking this change might be motivated something with a unique data file

Reply:
Prior to #654 the iteration through epochs was this:
```
epochs = nwbf.epochs.to_dataframe()
for epoch_index, epoch_data in epochs.iterrows():
    epoch_dict = dict()
    epoch_dict["nwb_file_name"] = nwb_file_name
    if epoch_data.tags[0]:
        epoch_dict["interval_list_name"] = epoch_data.tags[0]
    else:
        epoch_dict["interval_list_name"] = "interval_" + str(
            epoch_index
        )
    epoch_dict["valid_times"] = np.asarray(
        [[epoch_data.start_time, epoch_data.stop_time]]
    )
```
where epoch_data was a row of the dataframe and could be accessed like ` epoch_data.tags[0]`

In #654 it changed to 
```
epochs = nwbf.epochs.to_dataframe()
for _, epoch_data in epochs.iterrows():
    epoch_dict = {
        "nwb_file_name": nwb_file_name,
        "interval_list_name": epoch_data[1].tags[0]
        if epoch_data[1].tags
        else f"interval_{epoch_data[0]}",
        "valid_times": np.asarray(
            [[epoch_data[1].start_time, epoch_data[1].stop_time]]
        ),
    }
```
Here, epoch_data was still  row of the dataframe, but now the accessing calls changed to ` epoch_data[1].tags[0]`, which trys to get a tags property from the data in column [1] of the dataframe, causing an error.  

Maybe the intended change in that commit was:
```
epochs = nwbf.epochs.to_dataframe()
for epoch_data in epochs.iterrows():
    ....
```
In which case the changes in the other lines would run as intended

Reply:
Ok, the current edit ran fine for insertion, and the changes in there shouldn't go past this for loop so I think it's ok